caas/kubernetes/provider/k8swatcher_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package provider_test
+
+import (
+	jujuclock "github.com/juju/clock"
+	"github.com/juju/juju/caas/kubernetes/provider"
+	"github.com/juju/juju/core/watcher"
+
+	"k8s.io/client-go/tools/cache"
+)
+
+type KubernetesNotifyWatcherDummy struct {
```

Comment:
don't need this, we already have a mock notify watcher,
See core/watcher/watchertest/notify.go
NewMockNotifyWatcher